src/video_utils.py

```diff
@@ -174,6 +174,8 @@ def download_video(url):
     temp_dir = tempfile.mkdtemp()
     command = [
```

Comment:
probably something like:

.env.example
```
INSTACOOKIES=False
```
src/video_utils.py:
```
INSTACOOKIES = os.getenv('INSTACOOKIES', False)

command = [
          "yt-dlp",
          *(["--cookies", "instagram_cookies.txt"] if INSTACOOKIES else []),
          "-S",
          "vcodec:h264,fps,res,acodec:m4a",
           url,
```

Reply:
Could you clarify the goal for this? We could do that, but it changes nothing. 
`yt-dlp` with `--cookies` works fine with or without the file

Reply:
Ok, it's not checking if the file exists, only var value. 

Anyway, the base behavior would be the same

Reply:
> Could you clarify the goal for this? We could do that, but it changes nothing. `yt-dlp` with `--cookies` works fine with or without the file

I don't see a reason to include unused cli parameters if it can be avoided easily.